src/index.js

```diff
@@ -1,4 +1,5 @@
 /* @flow */
+import "babel-polyfill";
```

Comment:
I guess it is an attempt to fix the tests, also it doesn't so maybe you know what is missing.